.eslintrc.js

```diff
@@ -204,7 +204,7 @@ module.exports = {
         "no-useless-return": "error",
         "no-var": "error",
         "no-void": "error",
-        "no-warning-comments": "error",
+        "no-warning-comments": "off",
```

Comment:
The eslint file is new, so still turning things off that irk me while developing